docs/advanced_examples/DecisionTreeClassifier.ipynb

```diff
@@ -169,16 +182,15 @@
      "text": [
       "Number of test samples: 691\n",
       "Number of spams in test samples: 304\n",
-      "True Negative (legit mail well classified) rate: 0.9612403100775194\n",
-      "False Positive (legit mail classified as spam) rate: 0.03875968992248062\n",
+      "True Negative (legit mail well classified) rate: 0.9302325581395349\n",
+      "False Positive (legit mail classified as spam) rate: 0.06976744186046512\n",
```

Comment:
We lose a bit of accuracy here?

Reply:
does not seem very good indeed

Reply:
No idea why, I didn't change anything that would affect that.

We could probably seed notebooks

Reply:
reverted